src/amo/components/Page/index.js

```diff
@@ -75,7 +77,7 @@ export const PageBase = ({
 
   return (
     <div className="Page-amo">
-      <VPNPromoBanner />
+      {showVPNPromo ? <VPNPromoBanner /> : null}
```

Comment:
Is this working?

```js
{showVPNPromo && <VPNPromoBanner />}
```

Reply:
TBPH, I'm not sure when code like that works and when it does not. Sometimes code like that ends up outputting "0" or "false", because `false` && anything else evaluates to `false`. I guess in the case where `showVPNPromo` is `undefined` it would work, but in this case wouldn't it output "false", because `showVPNPromo` is `false`?

Reply:
Hmm, it does seem to work in this case. Is that because it works as long as it's a true boolean `false`, and the times it doesn't work is when the boolean expression evaluates to a number or string?

Reply:
I don't think I encountered issues with this approach so I don't know.